packages/@lwc/wds-core/src/node/plugins/mock-esm/handle/mock-resolved.js

```diff
@@ -6,9 +6,11 @@ const buildMockForResolved = (absPathToUnmockedOriginal, exportedNames) => `
 import * as __original__ from '${absPathToUnmockedOriginal}';
 
 ${GENERATED_MODULE_COMMENT}
-
 ${withoutDefault(exportedNames)
-  .map((name) => `export let ${name} = __original__['${name}'];`)
+  .map(
+    (name) => `
+  export let ${name} = __original__['${name}'];`,
```

Comment:
Why the new line-break here?

Reply:
I will remove that I added some code here and while removing forgot to remove new line break